src/backend.github.js

```diff
@@ -20,7 +20,7 @@ var _ = Mavo.Backend.register($.Class({
 			filename: `${this.mavo.id}${extension}`
 		};
 
-		this.info = _.parseURL(this.source, this.defaults);
+		this.info = _.parseURL(this.source, this.defaults, o);
```

Comment:
Do note there's another call to `parseURL` in `get()`. Do we need to modify that too?